src/main/webapp/loginpage.xhtml

```diff
@@ -126,43 +127,50 @@
                                         <label class="col-sm-4 control-label">
                                             #{bundle['auth.providers.title.shib']}
                                         </label>
-                                        <div id="idpSelect" class="col-sm-8">
-
+                                        
+                                        <!-- As of v6.7, an instance that uses Shibboleth as part of InCommon can switch to -->
+                                        <!-- using the new login workflow that relies on WayFinder and MDQ. -->
+                                        <div class="col-sm-8" jsf:rendered="#{LoginPage.isShibbolethUseWayFinder()}">
+                                            <h:outputLink styleClass="btn btn-default" value="#{LoginPage.getShibWayfinderRedirect()}">
+                                                <h:outputText value="Log In via InCommon"/>
```

Comment:
```suggestion
                                                <h:outputText value="Log In via Your Institution"/>
```

Credit to @jyuenger for this [idea](https://iqss.slack.com/archives/CVB2SMDFX/p1754921524812369).

Here's the existing button in context:

<img width="601" height="343" alt="Screenshot 2025-08-11 at 10 39 04 AM" src="https://github.com/user-attachments/assets/94110b19-7fa3-49bd-b8da-44287b6f91c6" />

Reply:
I did incorporate the new text for the button; and also removed the "**Your Institution**" text label. So this is the current look: https://demo.dataverse.org/loginpage.xhtml
<img width="618" height="315" alt="Screenshot 2025-09-02 at 5 29 00 PM" src="https://github.com/user-attachments/assets/3cbb471c-a22e-4279-943a-edbd93404b4b" />
Still need to add a sentence to the help text above the button.